src/Entity/Server.php

```diff
@@ -323,4 +327,34 @@ public function setHostingProvider(string $hostingProvider): self
 
         return $this;
     }
+
+    /**
+     * @return Collection<int, Installation>
+     */
+    public function getInstallations(): Collection
+    {
+        return $this->installations;
+    }
+
+    public function addInstallation(Installation $installation): self
```

Comment:
DocBlock?

Reply:
This is just getter/setter type code on a standard entity